### PR TITLE
Completes solution

### PR DESCRIPTION
**Pull request by janmilosh**

### CODE REVIEW

File: spec/refresher_spec.rb [hunk 1]
```diff
@@ -5,78 +5,78 @@
...
     end
 
     it "adds an item to the front of an array" do
-      expect([1]._fill_in_method_here_("banana")).to eq ["banana", 1]
+      expect([1].unshift("banana")).to eq ["banana", 1]
     end
 
     it "removes an item from the front of an array" do
-      expect([1, "banana"]._fill_in_method_here_).to eq ["banana"]
+      expect([1, "banana"].pop(1)).to eq ["banana"]
```
Comment (sjreich) [inline spec/refresher_spec.rb]: Hm.  There's something funny about the exercise.  I assume that the desired answer is `shift`, but that actually won't work, given the expected value.

Comment (sjreich) [inline spec/refresher_spec.rb]: Too funny.  So you actually already fixed this issue here: https://github.com/paircolumbus/RubyRefresher/commit/3138eef3d41ba95227bd7431ccc00bbb5a2a15a1 , and now you're submitting the results on the outdated version.

**Comment by sjreich** [comment]
Great work, Jan!

**Comment by janmilosh** [comment]
I'm not sure how I ended up with the outdated version. I changed this to the updated one.

**Pull request closed without merging**